app/presenters/taxon_organisations_presenter.rb

```diff
@@ -0,0 +1,65 @@
+class TaxonOrganisationsPresenter
+  ORG_PROMOTED_CONTENT_COUNT = 5
+
+  attr_reader :organisations
+
+  def initialize(organisations)
+    @organisations = organisations
+  end
+
+  def show_organisations?
+    organisations.any?
+  end
+
+  def show_more_organisations?
+    show_more_organisation_list.values.flatten.any?
+  end
+
+  def promoted_organisation_list
+    promoted_organisations_with_logos = organisation_list_with_logos.shift(ORG_PROMOTED_CONTENT_COUNT)
+    promoted_organisations_without_logos = organisation_list_without_logos.shift(ORG_PROMOTED_CONTENT_COUNT - promoted_organisations_with_logos.count)
+
+    {
+      "promoted_with_logos": promoted_organisations_with_logos,
+      "promoted_without_logos": promoted_organisations_without_logos
+    }
+  end
+
+  def show_more_organisation_list
+    {
+      "organisations_with_logos": organisation_list_with_logos.drop(ORG_PROMOTED_CONTENT_COUNT),
+      "organisations_without_logos": organisation_list_without_logos.drop(ORG_PROMOTED_CONTENT_COUNT - promoted_organisation_list[:promoted_with_logos].count)
+    }
+  end
+
+private
+
+  def organisations_with_logos
+    organisations.select(&:has_logo?)
+  end
+
+  def organisation_list_with_logos
+    organisations_with_logos.map do |org|
+      {
+
+          name: org.logo_formatted_title,
+          url: org.link,
+          brand: org.brand,
+          crest: org.crest,
+      }
+    end
+  end
+
+  def organisation_list_without_logos
+    orgs = organisations - organisations_with_logos
```

Comment:
A non-blocking comment: Maybe a good idea to rename orgs - it's not a very clear variable name.

Reply:
Changing it to `organisations_without_logos`. Will update and fixup.